assets/js/sketch.js

```diff
@@ -29,19 +29,36 @@ function setup() {
   rectMode(CORNER);
 }
 
+defInk = "#264180";//Default ink
```

Comment:
Specify the color in the comments as well. We want it to be more human-readable as well!